### DISCUSSION TITLE
Default mapping `gO` to `:Neorg toc` removed?

### DISCUSSION THREAD

**Discussion by Anrock** [nvim-neorg/neorg#1570] [Q&A] [answered]
Some time ago pressing `gO` (which is default keymap for "Show a filetype-specific, navigable "outline" of the current buffer") opened Neorg table of contents and now it doesn't.

I vaguely remember seeing patches reworking ToC and I suspect that keymapping got lost somewhere there. Was that intentional change or a bug?

If it was intentional - what is the recommended lua way to bind `gO` to `:Neorg toc` for .norg buffers specifically?

**Comment by DragoonSakura** [comment, marked as answer]
Assuming you are using **lua**.
Go to your nvim config directory (Usually `~/.config/nvim` in __linux__)
create directory (`~/.config/nvim/ftplugin`).

All file specific configs go inside ftplugin.
We first create a file for the desired filetype.
so create file `~/.config/nvim/ftplugtin/norg.lua`(so norg.lua for norg files).

__All__ configs here only applies to norg files.
Add: `vim.keymap.set( "n", "gO", "<cmd>Neorg toc<cr>", { desc = "Neorg table of contents" })`.

**Extra Note:** It is also recommended that you use `<localleader>` instead of `<leader>` in these filetype specific configs!

Good luck!
